src/scrapers/base-isracard-amex.ts

```diff
@@ -350,9 +350,9 @@ class IsracardAmexBaseScraper extends BaseScraperWithBrowser<ScraperSpecificCred
     this.page.on('request', (request) => {
       if (request.url().includes('detector-dom.min.js')) {
         debug('force abort for request do download detector-dom.min.js resource');
-        void request.abort();
+        void request.abort(undefined, 1000);
       } else {
-        void request.continue();
+        void request.continue(undefined, 10);
       }
     });
```

Comment:
Remove this on your way...

```suggestion
      if (request.url().includes('detector-dom.min.js')) {
        debug('force abort for request do download detector-dom.min.js resource');
        request.abort(undefined, 1000);
      } else {
        request.continue(undefined, 10);
      }
    });
```

I've never seen such syntax

Reply:
Adding `void` is to explicitly show we don't care about the returned promise and must be added here to satisfy the [@typescript-eslint/no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises) rule